server/odc-service/src/main/java/com/oceanbase/odc/service/onlineschemachange/oscfms/OscActionFsmBase.java

```diff
@@ -411,7 +414,7 @@ public ConnectionSession createConnectionSession() {
                     new DefaultConnectSessionFactory(connectionConfig).generateSession();
             ConnectionSessionUtil.setCurrentSchema(connectionSession,
                     dbName);
-            return new DefaultConnectSessionFactory(connectionConfig).generateSession();
+            return connectionSession;
```

Comment:
remember that the connection session should be expired when you don't use it anymore otherwise It will be lead to resource leak

Reply:
this is a wrapper, it will be called where the connection is needed and release connection after use